src/main/resources/assets/trinkets/lang/ru_ru.json

```diff
@@ -0,0 +1,20 @@
+{
+  "trinkets.tooltip.slots.any": "Можно надеть в §9любой§r слот для аксессуаров",
+  "trinkets.tooltip.slots.list": "Можно надеть в слоты для аксессуаров:",
+  "trinkets.tooltip.slots.single": "Можно надеть в слот для аксессуаров %d",
+  "trinkets.tooltip.attributes.all": "Когда надето:",
+  "trinkets.tooltip.attributes.single": "Когда надето в слот для аксессуаров %d:",
+  "trinkets.tooltip.attributes.slots": "%d Слотов",
+  "trinkets.slot.head.face": "Лица",
+  "trinkets.slot.head.hat": "Шляпы",
+  "trinkets.slot.chest.necklace": "Ожерелья",
+  "trinkets.slot.chest.back": "Спины",
+  "trinkets.slot.chest.cape": "Плаща",
+  "trinkets.slot.legs.belt": "Пояса",
+  "trinkets.slot.feet.aglet": "Эглета",
+  "trinkets.slot.feet.shoes": "Обуви",
+  "trinkets.slot.hand.glove": "Перчатки",
+  "trinkets.slot.offhand.glove": "Перчатки",
+  "trinkets.slot.hand.ring": "Кольца",
+  "trinkets.slot.offhand.ring": "Кольца"
```

Comment:
Just making sure, these are not used anywhere else besides `trinkets.tooltip.slots.single`? If so, they also probably shouldn't be capitalised.

Reply:
@emilyploszaj Are those used anywhere else besides tooltips? Im not sure

Reply:
These are the canonical "proper noun" names of slots, and currently I believe they're only used in tooltips. They're capitalized in english